pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -175,28 +176,33 @@ func createDefaultSvc(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Com
 	}
 	service.ObjectMeta.Annotations[constants.OpenshiftServingCertAnnotation] = componentMeta.Name + constants.ServingCertSecretSuffix
 
-	if val, ok := componentMeta.Annotations[constants.ODHKserveRawAuth]; ok && strings.EqualFold(val, "true") {
-		httpsPort := corev1.ServicePort{
-			Name: "https",
-			Port: constants.OauthProxyPort,
-			TargetPort: intstr.IntOrString{
-				Type:   intstr.String,
-				StrVal: "https",
-			},
-			Protocol: corev1.ProtocolTCP,
-		}
-		ports := service.Spec.Ports
-		replaced := false
-		for i, port := range ports {
-			if port.Port == constants.CommonDefaultHttpPort {
-				ports[i] = httpsPort
-				replaced = true
+	if resourceType == constants.InferenceGraphResource {
+		port443 := int32(443)
```

Comment:
Do we need to separate this into 2 lines?
```
servicePorts[0].Port = int32(443)
```
would it be ok?

Reply:
I changed it. It works as a one line.